parsing_scripts/output.json

```diff
@@ -0,0 +1,49 @@
+{
+  "account": {
+    "name": "Anastasia Monosienco",
+    "balance": 2301.73,
+    "currency": "MDL"
+  },
+  "transactions": [
+    {
+      "date": "1 mie. mai 2019 09:07",
+      "description": "Moldcell",
+      "amount": 102
+    },
+    {
+      "date": "30 mar. aprilie 2019 00:00",
+      "description": "Cl Deposit Int Due Special",
+      "amount": 0.89
+    },
+    {
+      "date": "26 vin. aprilie 2019 10:22",
+      "description": "Moldcell",
+      "amount": 50
+    },
+    {
+      "date": "24 mie. aprilie 2019 18:28",
+      "description": "Orange",
+      "amount": 30
+    },
+    {
+      "date": "18 joi aprilie 2019 10:48",
+      "description": "Plata retail GRANIER A. IULIA CHISINAU Moldova, Rep. Of",
+      "amount": 35
+    },
+    {
+      "date": "16 mar. aprilie 2019 14:57",
+      "description": "Plata retail MAIB GREEN HILLS MARKE Chisinau      Moldova, Rep. Of",
```

Comment:
Please normalize the string. Make it look clean, w/o extra whitespaces or special symbols if such exist.